mmv1/products/compute/api.yaml

```diff
@@ -3285,15 +3285,15 @@ objects:
           are 4096 and 16384, other sizes may be added in the future.
           If an unsupported value is requested, the error message will list
           the supported values for the caller's project.
-      - !ruby/object:Api::Type::Enum
+      - !ruby/object:Api::Type::String
         name: 'interface'
         min_version: 'beta'
+        # interface is removed using url_param_only to preserve schema definition
+        # and prevent sending or reading in API requests
+        url_param_only: true
+        deprecation_message: This field is no longer in use, disk interfaces will be automatically determined on attachment.
```

Comment:
What action can a user take to resolve the deprecation message?

Reply:
Added note to remove it.

Added a DSF to always suppress diffs on this field, allows users to remove this if they had specified NVME without adding ignore_diffs